pkgs/applications/networking/instant-messengers/nheko/default.nix

```
@@ -69,8 +83,13 @@ mkDerivation rec {
       libnice
     ]);
 
+  preConfigure = ''
+    substituteInPlace CMakeLists.txt --replace "# Fixup bundled keychain include dirs" "find_package(Boost COMPONENTS iostreams system  thread REQUIRED)"
```

The line is too long
```suggestion
    substituteInPlace CMakeLists.txt --replace \
      "# Fixup bundled keychain include dirs" \
      "find_package(Boost COMPONENTS iostreams system  thread REQUIRED)"
```